src/frogger.cpp

```diff
@@ -48,21 +58,28 @@ leapfrog::TensorMap1<int> get_age_groups_hiv_span(const Rcpp::List projection_pa
 Rcpp::List run_base_model(const Rcpp::List data,
                           const Rcpp::List projection_parameters,
                           SEXP sim_years,
+                          SEXP hiv_steps_per_year,
                           std::string hiv_age_stratification = "full") {
   const int proj_years = get_simulation_years(data, sim_years);
+  const int hiv_steps = get_hiv_steps_per_year(hiv_steps_per_year);
   const int num_genders = 2;
   const int age_groups_pop = 81;
   const int fertility_first_age_group = 15;
   const int age_groups_fert = 35;
   const int disease_stages = 7;
   const int treatment_stages = 3;
   const int hiv_adult_first_age_group = 15;
+  const int adult_incidence_first_age_group = hiv_adult_first_age_group;
+  // Hardcoded 15-49 for now
+  const int pAG_INCIDPOP = 35;
   // 0-based indexing vs R 1-based
   const int time_art_start =
       Rcpp::as<int>(projection_parameters["t_ART_start"]) - 1;
   auto age_groups_hiv_span =
       get_age_groups_hiv_span(projection_parameters, hiv_age_stratification);
   int age_groups_hiv = static_cast<int>(age_groups_hiv_span.size());
+  const int scale_cd4_mortality =
+      Rcpp::as<int>(projection_parameters["scale_cd4_mort"]);
```

Comment:
Consider helpers with better errors (do this in another ticket) https://github.com/mrc-ide/dust/blob/master/inst/include/dust/r/helpers.hpp#LL19C1-L41C1